apps/studio/src/features/editing-experience/components/EditLinkPreview.tsx

```diff
@@ -38,43 +38,49 @@ export const EditCollectionLinkPreview = (): JSX.Element => {
     siteId,
   })
 
-  const parentPermalink = permalink.split("/").slice(0, -1).join("/")
-  const parentTitle = parent?.title || ResourceType.Collection
+  const parentPermalink = useMemo(
+    () => permalink.split("/").slice(0, -1).join("/"),
+    [permalink],
+  )
+  const parentTitle = useMemo(
+    () => parent?.title || ResourceType.Collection,
+    [parent?.title],
+  )
 
-  const siteMap = {
-    id: "root",
-    permalink: "/",
-    lastModified: "2024-10-14T07:05:08.803Z",
-    layout: "homepage",
-    title: "An Isomer Site",
-    summary: "",
-    children: [
-      {
-        id: "collection",
-        permalink: parentPermalink,
-        lastModified: "02 May 2023",
-        layout: "collection",
-        title: parentTitle,
+  const siteMap = useMemo(
+    () =>
+      ({
+        id: "root",
+        permalink: "/",
+        lastModified: "2024-10-14T07:05:08.803Z",
+        layout: "homepage",
+        title: "An Isomer Site",
         summary: "",
-        collectionPagePageProps: { tagCategories },
         children: [
           {
-            id: "9999999",
-            title,
-            image,
-            date,
-            ref,
-            summary: summary ?? "",
-            layout: "link",
-            permalink,
-            lastModified: new Date().toString(),
-            category,
-            tagged,
+            id: "collection",
+            permalink: parentPermalink,
+            lastModified: "02 May 2023",
```

Comment:
nit: I think we should take the chance to change all the `lastModified` to use the `currentDate`. The dates used here seems very arbitrary.

Reply:
sounds good, done